_posts/2018-04-20-totw-24.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Tip of the Week #24: Copies, Abbrv."
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/24
+type: markdown
+order: "024"
+---
+
+Originally posted as TotW #24 on Nov 26, 2012
+
+*by Titus Winters, [(titus@google.com)](mailto:titus@gmail.com) and
+Chandler Carruth [(chandlerc@google.com)](mailto:chandlerc@google.com)*
+
+*"To copy others is necessary, but to copy oneself is pathetic." - Pablo
+Picasso*
+
```

Comment:
This also should refer to tips 55 and 77 ... or just skip since those are better explanation of this material.

Reply:
Done